projects/dotcms-field-elements/src/components/dot-tags/dot-tags.tsx

```diff
@@ -29,6 +23,9 @@ export class DotTagsComponent {
     /** Value formatted splitted with a comma, for example: tag-1,tag-2 */
     @Prop({ mutable: true, reflectToAttr: true }) value = '';
 
+    /** Name that will be used as ID */
+    @Prop() data: () => Promise<string[]>;
```

Comment:
really, if the user need to use this component out the dot-form need to pass this functions always, I think it should have a default value maybe

Reply:
No, because what we need to publish in the library is a tag field that can support autocomplete form X source, that X will be given for who is using it.